src/libs/actions/Policy/Category.ts

```diff
@@ -638,6 +650,7 @@ function renamePolicyCategory(policyID: string, policyCategory: {oldName: string
                     rules: {
                         approvalRules,
                     },
+                    mccGroup: updatedMccGroupWithClearedPendingAction,
```

Comment:
Shouldn't this have the original mccGroup and not the updated one given we failed to update it?

Reply:
@youssef-lr I've updated the MCC group data for the failure data.